src/main/java/ysoserial/exploit/JSF.java

```diff
@@ -0,0 +1,69 @@
+package ysoserial.exploit;
+
+
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLEncoder;
+
+import org.apache.commons.codec.binary.Base64;
+
+import ysoserial.payloads.ObjectPayload.Utils;
+
+
+/**
+ * @author mbechler
+ *
+ */
+public class JSF {
```

Comment:
A description of how this works would be ideal here, especially if/how it interoperates with other ysoserial components.